modules/introduction/partials/new-features-72.adoc

```diff
@@ -24,3 +24,6 @@ See xref:rest-api:rest-bucket-create.adoc[Creating and Editing Buckets].
 
 * The tools `cbbackupmgr`, `cbimport`, and `cbexport` can now specify that an alternate address be used, using the `network=external` option, when connecting from an external network.
 See *Host Formats* information in xref:backup-restore:backup-restore.adoc[cbbackupmgr], xref:tools:cbimport.adoc[cbimport], and xref:tools:cbexport.adoc[cbexport].
+
+* New metrics are provided for tracking XDCR _conflict resolution_.
```

Comment:
* A new metric, kv_conflicts_resolved, is provided for tracking XDCR _conflict resolution_ on the target cluster.